### PR TITLE
Useful Links integration

### LINKED ISSUE #3927: Use `useful links` link promo in Simorgh
**Is your feature request related to a problem? Please describe.**
Use `useful link` link promo created in https://github.com/bbc/psammead/issues/2222 in Simorgh, and add it to the whitelist.

```
{
"name": "Guide promo",
"indexImage": {
"id": "63692548",
"subType": "index",
"href": "http://b.files.bbci.co.uk/14A31/test/_63692548_000327537-1.jpg",
"path": "/cpsdevpb/14A31/test/_63692548_000327537-1.jpg",
"height": 549,
"width": 976,
"altText": "A lone Koala perches in a eucalyptus tree",
"caption": "Koalas are from Australia",
"copyrightHolder": "BBC"
},
"uri": "http://www.bbc.com/azeri",
"contentType": "Guide",
"assetTypeCode": "PRO",
"timestamp": 1569321103000,
"type": "link"
},
```

**Describe the solution you'd like**
Using the useful links promo in Simorgh. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: This will require unit and Cypress tests

- [ ] This feature is expected to need manual testing.

**Additional context**
See WSFRONTPGS-226 for details on the expected contentType.

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #3927 

**Overall change:** 
Integrated "Useful Links" component in Simorgh.

**Code changes:**
- Updated Igbo fixture data to include useful links of type 'Guide'
- Added 'Guide' content type to whitelist
- Imported Useful Links component from Psammead
- Added Useful Links to Front Page Section
- Added a story to Simorgh storybook

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by OlgaLyubin** [comment]
This PR is basically a copy of https://github.com/bbc/simorgh/pull/4114 which is now closed (due to some issues with GPG keys I was not able to push commits there anymore)

**Comment by OlgaLyubin** [comment]
Previous code reviews:

Screenshot 2019-10-15 at 13 55 28
_____________
Screenshot 2019-10-15 at 13 55 38
_____________
Screenshot 2019-10-15 at 13 55 58

File: src/app/containers/FrontPageSection/index.jsx [hunk 1]
```diff
@@ -125,28 +158,14 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
...
-            item={items[0]}
-            sectionNumber={sectionNumber}
-            storyNumber={0}
-          />
-        </MarginWrapper>
+        <StoryPromoRenderer
+          items={items}
+          firstSection={isFirstSection}
+          sectionNumber={sectionNumber}
+        />
```
Comment (AlistairGempf) [inline src/app/containers/FrontPageSection/index.jsx:168]: Is this logic tested in the tests? If not we should check that the container renders useful links when the strapline name is `Useful links`.

**Review by hotinglok — APPROVED** [review]
LGTM

File: scripts/bundleSizeConfig.js [hunk 2]
```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 547,
```
Comment (dr3) [inline scripts/bundleSizeConfig.js]: Does the size of the min not increase too?

Comment (rhenshaw56) [reply scripts/bundleSizeConfig.js]: I think it does
Screen Shot 2019-10-16 at 11 55 25

**Review by DenisHdz — COMMENTED** [review]
Shall the `UsefulLinks` folder be located outside the `FrontPageSection`?

**Comment by OlgaLyubin** [comment]
> Shall the `UsefulLinks` folder be located outside the `FrontPageSection`?

I could put them outside the `FrontPageSection` if needed. For now I put them inside that folder because it seemed to me that `UsefulLinks` are integrated into `FrontPageSection` the same way as e.g. `IndexAlsos` are integrated into `StoryPromo` and hence need to be inside of a folder.

File: scripts/bundleSizeConfig.js (hunk 2 above)
Comment (OlgaLyubin) [reply scripts/bundleSizeConfig.js]: Good point, thank you

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by DenisHdz — APPROVED** [review]
Great work on the integration 👍

**Review by rhenshaw56 — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good! 👍

**Comment by jamesbrumpton** [comment]
lgtm

**Comment by OlgaLyubin** [comment]
Moved to PR in progress for a small tweak on IDs

**Pull request merged**